src/Ledger/Utxo.lagda

```diff
@@ -376,6 +444,14 @@ to \consumed or \produced depending on its sign. This is done via
 \negPart and \posPart, which satisfy the key property that their
 difference is the identity function.
 
+Figures~\ref{fig:functions:utxo} also shows the signature of an inductive type
```

Comment:
I don't like the mention of 'inductive type' here. It's an inductive proof, so you won't be able to understand this unless you understand 'propositions as types'. We avoid this elsewhere, so I think it could be avoided here as well.